src/lspsrc/exthuk.34

```diff
@@ -0,0 +1,121 @@
+;;;   EXTHUK			-*-Mode:Lisp;Package:SI;Lowercase:T-*-
+;;;   ****************************************************************
+;;;   *** MACLISP **** EXTended datatype scheme, compiler helper *****
+;;;   ****************************************************************
+;;;   ** (c) Copyright 1981 Massachusetts Institute of Technology ****
+;;;   ****************************************************************
+
+(herald EXTHUK /33)
```

Comment:
/34 or just (herald EXTHUK)?

Reply:
Good catch.  I didn't change the source -- I just included the one found on backups.  There is an issue of faithfulness here, but in this case, we can either a) update the herald in this commit, or b) add an issue to fix the heralds in various files where they are wrong -- introducing a new version (with corrected herald).  Otherwise, we're changing an original source without updating the FN2 (version) and can set ourselves up for confusion when "our" version N is not the same as the historical version N.  Thoughts?